src/chapters/spring/mvc.rst

```diff
@@ -0,0 +1,28 @@
+Design Patterns, MVC, and Spring, Oh My!
+========================================
+
+.. index:: ! design pattern, ! Model-View-Controller, ! MVC, ! model, ! view, ! controller
+
+So far, we have been designing our applications by diagrammming classes, drawing connections, and adding in interfaces.
+This practice benefits us because we can start seeing issues *before* we start coding.
+Many software developers start their applications with this process.
+But before we start diagramming our ``Cat`` class and our ``HouseCat`` class, we decide on the template for our design that we want to use.
```

Comment:
I'd try removing the 'but' at that start of this sentence, or maybe replace it with 'and'

Reply:
Done! Thanks!